javascript/new-src/Gruntfile.js

```diff
@@ -0,0 +1,85 @@
+/* jslint node: true */
+"use strict";
+module.exports = function (grunt) {
+ grunt.initConfig({
+  pkg: grunt.file.readJSON('package.json'),
+
+  bower: {
+   dev: {
+    options: {
+     targetDir: 'javascript/new-src/src/libs',
+     install: true,
+     copy: true,
+     cleanup: true,
+     layout: 'byComponent'
+    }
+   }
+  },
+
+  uglify: {
+   default: {
+    files: {
+     'olad/www/new/js/app.min.js': ['javascript/new-src/src/js/app.js']
+    },
+    options: {
+     mangle: true,
+     sourceMap: true,
+     sourceMapName: 'olad/www/new/js/app.min.js.map',
+     banner: "/**\n" +
```

Comment:
You can ditch the banner for the minified version I think ( @nomis52 ?), we've not bothered with it in the past.